lisp/ess-inf.el

```diff
@@ -2246,7 +2246,7 @@ START-ARGS gets passed to the dialect-specific
       ;; Use current working directory as default
       (let ((project-find-functions nil)
             (ess-directory-function nil)
-            (ess-startup-directory (ess-get-working-directory))
+            (ess-startup-directory default-directory)
```

Comment:
The scoping isn't correct here I think. We don't want to change the directory of the process to that of the current buffer.

```suggestion
            (ess-startup-directory (ess-get-process-variable 'default-directory))
```

Reply:
Thanks, updated.